File: web-ui/src/displays/components/Text.js
import React from "react";
import { withStyles } from '@material-ui/core/styles';
import AbstractText from "../../../gen/displays/components/AbstractText";
import TextNotifier from "../../../gen/displays/notifiers/TextNotifier";
import TextRequester from "../../../gen/displays/requesters/TextRequester";
import DisplayFactory from 'alexandria-ui-elements/src/displays/DisplayFactory';

const styles = theme => ({});

class Text extends AbstractText {

	constructor(props) {
		super(props);
		this.notifier = new TextNotifier(this);
		this.requester = new TextRequester(this);
	};


}

export default withStyles(styles, { withTheme: true })(Text);
DisplayFactory.register("Text", withStyles(styles, { withTheme: true })(Text));

File: framework/ui/src/io/intino/alexandria/ui/displays/components/Text.java
package io.intino.alexandria.ui.displays.components;

import io.intino.alexandria.core.Box;
import io.intino.alexandria.ui.displays.notifiers.TextNotifier;

public class Text<DN extends TextNotifier, B extends Box> extends AbstractText<DN, B> {

    public Text(B box) {
        super(box);
    }

}